### ISSUE TITLE
Add a table

### ISSUE THREAD

**Issue by SpaceboyRoss01** [os-js/osjs-gui#5]
Having a dedicated table component would be nice.

**Comment by andersevenrud** [comment]
![os js table](https://user-images.githubusercontent.com/161548/49476540-8e988b00-f81a-11e8-8a3e-4bf3805a3714.png)

I think it will be something like this.

Example usage in this case:

```javascript
            h(Table, {
              box: {
                grow: 1
              },
              zebra: true,
              columns: ['a', 'b', 'c'],
              rows: [
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3']
              ],
              footer: ['x', 'y', 'z']
            })
```

**Comment by SpaceboyRoss01** [comment]
Yeah, that'll be good but could there be support for the filter buttons on the header?04.12.2018, 14:16, "Anders Evenrud" <notifications@github.com>:
I think it will be something like this.
Example usage in this case:
            h(Table, {
              box: {
                grow: 1
              },
              zebra: true,
              columns: ['a', 'b', 'c'],
              rows: [
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3'],
                ['1', '2', '3']
              ],
              footer: ['x', 'y', 'z']
            })

**Comment by andersevenrud** [comment]
Sure, that could be added.
